scripts/auth.ps1

```diff
@@ -0,0 +1,46 @@
+param (
+    [string]$server = "https://localhost:5001"

```

Comment:
It would be nice to include Update-Help script line in the beginning of script scope code

Reply:
In my opinion it's unnecessary, misleading and out of scope. It can cause errors if you don't have administrator privileges.

Reply:
The Update-Help is nice to have feature, which is nice to have. Lets add than possibility for user to choose, does he actually want to update it?

Reply:
I can do this but I'm still against it. In my opinion it's unnecessary. Every time user runs this script it'll ask him if he wants to update help? That's stupid innit bruv. Again in case user doesn't have administrator privileges it'll throw an error and whole script won't work because of that. After a little bit of investigation I found out there is no way to track if help files are up to date so we can't even notify user about that either.

Reply:
It makes sense, lets not include it into the script.